### PR TITLE
Closes #1508: Use specific otel-collector image instead of latest

### LINKED ISSUE #1508: [Bug] - Exporters Integration Tests fail
**Describe the bug**
Integration tests involving the [opentelemetry-java/otel-collector](https://github.com/orgs/open-telemetry/packages/container/package/opentelemetry-java%2Fotel-collector) fail.

**To Reproduce**
1. Start docker.
2. Run tests in core/src/test/.../exporter

### PR DESCRIPTION
**Pull request by aaronweissler**
Closes #1508

- - -
This change is [Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)

### CODE REVIEW

File: inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
 @Testcontainers(disabledWithoutDocker = true)
 abstract class ExporterServiceIntegrationTestBase extends SpringTestBase {
 
-    static final String COLLECTOR_IMAGE = "ghcr.io/open-telemetry/opentelemetry-java/otel-collector";
+    static final String COLLECTOR_IMAGE = "ghcr.io/open-telemetry/opentelemetry-java/otel-collector@sha256:d34519458388e55a3fce38a33e6bc424267c1f432927c09e932ba45f7575bd84";
```
Comment (aaronweissler) [inline inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java:60]: The issue was that not specifying any specific image leads to using the `latest` one which has been updated recently, which apparently broke our tests. Question is, does our implementation have to work with `latest` or is it enough to only work with the older one?

**Review by heiko-holz — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)* status: all files reviewed, 1 unresolved discussion (waiting on @heiko-holz)

**Review by heiko-holz — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)* status: all files reviewed, 1 unresolved discussion (waiting on @heiko-holz)

**Review by heiko-holz — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)* status: all files reviewed, 1 unresolved discussion (waiting on @heiko-holz)

**Review by heiko-holz — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)* status: all files reviewed, 1 unresolved discussion (waiting on @aaronweissler)

___
*[`inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java` line 60 at r1](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509#-N9W0jskNzxq8C9inic--r1-60:-N9_XihpDuaUyX7O5gbd:bgwsmq6) ([raw file](https://github.com/inspectit/inspectit-ocelot/blob/649bbb1012e28f93182ec9da47c691a6a6463597/inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java#L60)):*
<details><summary><i>Previously, aaronweissler wrote…</i></summary><blockquote>

The issue was that not specifying any specific image leads to using the `latest` one which has been updated recently, which apparently broke our tests. Question is, does our implementation have to work with `latest` or is it enough to only work with the older one?
</blockquote></details>

Thanks for spotting this!

**Review by heiko-holz — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509)* status: all files reviewed, 1 unresolved discussion (waiting on @aaronweissler)

___
*[`inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java` line 60 at r1](https://reviewable.io/reviews/inspectit/inspectit-ocelot/1509#-N9W0jskNzxq8C9inic--r1-60:-NAhnr6L2G4IAeGvlfx7:b-hv95np) ([raw file](https://github.com/inspectit/inspectit-ocelot/blob/649bbb1012e28f93182ec9da47c691a6a6463597/inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java#L60)):*
<details><summary><i>Previously, heiko-holz (Heiko Holz) wrote…</i></summary><blockquote>

Thanks for spotting this!
</blockquote></details>

The issue was that from `v0.44.0` to `v0.45.0` the `otlp` exporter uses gzip compression by default (see https://github.com/open-telemetry/opentelemetry-collector/releases?q=0.45.0&expanded=true).

The version will be upgraded with a fix in another PR

**Pull request merged**